### DISCUSSION TITLE
Regarding Velocity sampling.

### DISCUSSION THREAD

**Discussion by USMANSHAUKAT001** [turbinesFoam/turbinesFoam#346] [Q&A]
Hi Pete, 
I am a master's student working on the simulation of wind turbines using this library. I want to know at which point the velocity is sampled. Is it at the middle of the actuator segment or the quarter chord position?

**Comment by petebachant** [comment]
The sampling is done at the `position_` of the `actuatorLineElement`, which is the quarter chord: https://github.com/turbinesFoam/turbinesFoam/wiki/Actuator-line-geometry

It looks like there's a comment in the code that claims this to be the half-chord location, but that's incorrect.

**Comment by USMANSHAUKAT001** [comment]
Thank you for the respone Pete.
I want to sample the velocity at center not at the the quarter chord. Do you have a suggestion how can I change it?

**Comment by petebachant** [comment]
You would need to add an option for this in the `actuatorLineElement` class, or you can move around the blade mounting offset in the geometry definition in `fvOptions`, though that will also move around the center of where the force is applied to the flow field. Do you want the sampling point to be different from the one at which the force is applied?

**Reply by USMANSHAUKAT001** [reply]
Hi again pete,
I have basically applied the methadology of Jha et al. for the equivalent elliptic planform for gaussian force width. However, the result didn't showed the effect of reduce loading at the tip of the blade which can be observed in the paper. Therefore, I am wondering maybe it is because of the velocity sampling point. In the paper Jha et al, mentioned that the velocity is sampled at the center of actuator line element . As stated in paper "local velocity is sampled directly at the center of each actuator line element. Because the velocity is sampled at the center of each actuator element, which is the center of the bound vortex circulating about the actuator line, the effects of the upwash and downwash created by the bound vortex are not seen". 
Therefore, i want to sample the velocity at the center of the element.
